components/brave_wallet/browser/eth_abi_decoder_unittest.cc

```diff
@@ -61,7 +61,7 @@ TEST(EthABIDecoderTest, ABIDecodeUint256) {
       "0x00000000000000000000000000000000000000000000000000000000000000ff"
       "ff",
       &data));
-  ASSERT_FALSE(ABIDecode({"uint256"}, data));
+  ASSERT_TRUE(ABIDecode({"uint256"}, data));
```

Comment:
Can we compare the actual return value here for correctness too pls?

Reply:
Fixed here: 40cd5d0e8b5c85b04384a2248e69e7e6cca639bc, and in all other occurrences.